feature-wordflow/src/main/java/ru/coderedwolf/wordlearn/wordflow/presentation/CreateWordFeature.kt

```diff
@@ -18,25 +20,26 @@ import ru.coderedwolf.wordlearn.word.model.WordExample
 import ru.coderedwolf.wordlearn.wordflow.presentation.CreateWordFeature.*
 import javax.inject.Inject
 
+@PerFragment
 class CreateWordFeature @Inject constructor(
-        categoryId: Long,
-        router: FlowRouter,
-        schedulerProvider: SchedulerProvider,
-        wordInteractor: WordRepository
+    categoryId: Long,
+    router: FlowRouter,
+    schedulerProvider: SchedulerProvider,
+    wordInteractor: WordRepository
```

Comment:
May we rename 'wordInteractor' to' wordRepository'?

Reply:
done